GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -623,6 +623,7 @@ private void writeAnswer(
             boolean useQuestionId) {
 
         assert value != null;
+        value = value.trim();
```

Comment:
Good catch. This was accidentally reintroduced when on `git merge`. I'll remove it